nftables-slim.yaml

```
@@ -0,0 +1,58 @@
+package:
+  name: nftables-slim
+  version: "1.1.3"
+  epoch: 0
+  description: Netfilter tables userspace tools without interactive CLI
+  copyright:
+    - license: GPL-2.0-or-later
+  dependencies:
+    runtime:
+      - merged-usrsbin
+      - wolfi-baselayout
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - bison
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - flex
+      - gmp-dev
+      - libmnl-dev
+      - libnftnl-dev
+      - py3-gpep517
+      - py3-setuptools
+      - py3-wheel
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 9c8a64b59c90b0825e540a9b8fcb9d2d942c636f81ba50199f068fde44f34ed8
+      uri: https://netfilter.org/projects/nftables/files/nftables-${{package.version}}.tar.xz
+
+  - uses: autoconf/configure
+    with:
+      opts: |
+        --sbindir=/usr/bin --without-cli --disable-man-doc --disable-debug --without-PACKAGE
```

The last flag is extraneous:

```suggestion
        --sbindir=/usr/bin --without-cli --disable-man-doc --disable-debug
```